spec/features/lettings_log_spec.rb

```diff
@@ -286,6 +286,10 @@
     context "when reviewing a complete log" do
       let(:lettings_log) { create(:lettings_log, :completed) }
 
+      before do
+        lettings_log.owning_organisation.update!(name: "MHCLG")
```

Comment:
It looks like the test doesn't actually need this - does it work to remove it? (might be the same for the next few files too)

Reply:
Ah, yep, missed them!